examples/PrivateLink/main.tf

```diff
@@ -1,7 +1,7 @@
 terraform {
   required_providers {
     clickhouse = {
-      version = "0.0.5"
+      version = "0.0.10"
```

Comment:
technically this example won't work until this PR is merged, and thus released with a future version of the provider, right?

Reply:
This is the current version so we are all good here.